src/js/ResponsiveNav.js

```diff
@@ -75,6 +79,27 @@ function ResponsiveNav(rootEl) {
 		}
 	}
 
+	function incrementMenuDepths(el) {
+		// id's are prefixed to ensure that any id based functionality uses the visible element
+		// for example a 'label' tag with a 'for' attribute will not find the correct input it
+		// relates to as it uses the first matching id in the document
+		let child;
+		if (el.hasChildNodes()) {
+			const children = el.childNodes;
+			for (let i = 0, l = children.length; i < l; i++) {
+				child = children[i];
+				if (child instanceof HTMLElement) {
+					if (child.hasAttribute('data-o-hierarchical-nav-level')) {  
+                        // increment level
+                        let oldLevel = isNaN(parseInt(child.getAttribute('data-o-hierarchical-nav-level'))) ? 0 : parseInt(child.getAttribute('data-o-hierarchical-nav-level'))              
```

Comment:
Indentation and might be worth taking this out to a separate variable: `parseInt(child.getAttribute('data-o-hierarchical-nav-level'))`